tools/docker/Dockerfile.marvin

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#
+# build for cloudstack_home_dir not this folder
+FROM python:2
+
+MAINTAINER "Apache CloudStack" <dev@cloudstack.apache.org>
+LABEL Vendor="Apache.org"
+LABEL License=ApacheV2
+LABEL Version=4.6.0
+
+ENV PKG_URL=http://jenkins.buildacloud.org/job/cloudstack-marvin-master/lastSuccessfulBuild/artifact/tools/marvin/dist/Marvin-4.6.0-SNAPSHOT.tar.gz
+
+RUN pip install --allow-external mysql-connector-python mysql-connector-python
+RUN pip install ${PKG_URL}
+
+RUN mkdir -p /marvin
+COPY setup/dev /marvin/dev
+COPY tools/marvin/marvin /marvin/marvin
+COPY test/integration /marvin/integration
+
+WORKDIR /marvin
```

Comment:
so you want to get a shell into the container everytime ?

    $ docker run -ti marvin
   34543534#

something like that ?
Because you will not use the same scripts everytime ?